share/completions/quilt.fish

```diff
@@ -1,35 +1,35 @@
-complete -c quilt -a grep -r --no-file -d "Grep through the source files, recursively, skipping patches and quilt meta-information"
-complete -c quilt -a header -r --no-file -d " Print or change the header of the topmost or specified patch"
+complete -c quilt -a delete -r -d "Remove the topmost/given patch from the series file"
+complete -c quilt -a diff -r -d "Produces a diff of the given file(s) in the topmost/given patch"
+complete -c quilt -a edit -r -d "Edit the given file(s) in $EDITOR"
+complete -c quilt -a files -r -d "Lists files that the topmost/given patch changes"
+complete -c quilt -a fold -r -d "Integrate the patch read from standard input into the topmost patch"
+complete -c quilt -a fork -r --no-file -d "Fork the topmost patch"
+complete -c quilt -a graph -r --no-file -d "Generates a dot(1) directed graph of dependencies between applied patches"
+complete -c quilt -a grep -r --no-file -d "Grep through source files recursively (skips patches and quilt meta-information)"
+complete -c quilt -a header -r --no-file -d "Print/change the header of the topmost/given patch"
 complete -c quilt -a import -r -d "Import external patches"
-complete -c quilt -a mail -r -d "Create  mail  messages  from all patches in the series file, and either store or send them"
-complete -c quilt -a new -r -d "Create a new patch with the specified file name, and  insert  it after the topmost patch"
-complete -c quilt -a next -r -d "Print  the  name  of  the next patch after the specified or topmost patch in the series file"
-complete -c quilt -a patches -r -d "Print the list of patches that modify the specified file"
-complete -c quilt -a pop -r -d "Remove  patch(es) from the stack of applied  patches"
-complete -c quilt -a previous -r -d "Print the name of the previous patch before the specified or topmost patch in the series file"
+complete -c quilt -a mail -r -d "Create mail messages from patches in the series file, stores/sends them"
+complete -c quilt -a new -r -d "Creates new patch with the given name, inserts it after topmost patch"
+complete -c quilt -a next -r -d "Prints name of patch after the topmost/given patch in the series file"
+complete -c quilt -a patches -r -d "Print the list of patches that modify the given file"
+complete -c quilt -a pop -r -d "Remove patch(es) from the stack of applied patches"
+complete -c quilt -a previous -r -d "Prints name of patch before the topmost/given patch in the series file"
 complete -c quilt -a push -r -d "Apply patch(es) from the series file"
-complete -c quilt -a refresh -r -d "Refreshes the specified patch, or the topmost patch by default"
-complete -c quilt -a remove -r -d "Remove one or more files from the topmost or named patch"
-complete -c quilt -a rename -r -d "Rename the topmost or named patch"
+complete -c quilt -a refresh -r -d "Refreshes the given patch, or the topmost patch by default"
```

Comment:
We use the imperative mood - "Refresh the given patch...". This consistently saves characters ("refresh" vs "refreshes")

Reply:
(this goes for the rest as well!)